include/graphblas/ops.hpp

```diff
@@ -1142,6 +1166,11 @@ namespace grb {
 		static const constexpr bool value = true;
 	};
 
+	template< class Op >
+	struct is_idempotent< operators::logical_not< Op >, void > {
+		static const constexpr bool value = is_idempotent< Op >::value;
```

Comment:
This is also correct -- if given the same value to both operands returned that same value, then also that same value is negated and idempotency is retained. Please also note this in an (internal) comment above this class def.